libs/lib_loaders.py

```diff
@@ -31,7 +31,7 @@
 
 class DotHeaders:
 
-    version = "1.2.8-DEV.1"
+    version = "1.2.9-DEV.0"
```

Comment:
uh why change this? here did literally anything else change in net?

Reply:
dotheaders manages gcc compiling it
it wont recompile and use the new binary unless the version changes